java-manta-benchmark/src/main/java/com/joyent/manta/benchmark/Benchmark.java

```diff
@@ -45,6 +45,7 @@
  * Manta performance from the command line.
  *
  * @author <a href="https://github.com/dekobon">Elijah Zupancic</a>
+ * @author <a href="https://github.com/nairashwin952013">Ashwin A Nair</a>
```

Comment:
If you are making only code style changes, you might not want to sign your name on it.

Reply:
Noted. I had introduce more and then deleted them will keep that in mind.